textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/utils/Utils.java

```diff
@@ -118,7 +126,14 @@ public static ITuple getSpanTuple( List<IField> fieldList, List<Span> spanList,
      */
     public static Schema createSpanSchema(Schema schema) {
         List<Attribute> dataTupleAttributes = schema.getAttributes();
-        //spanAttributes contains all attributes of dataTupleAttributes and an additional SPAN_LIST_ATTRIBUTE
+        // If SourceOperator is of type Keyword, spanschema is already provided

```

Comment:
@zuozhi Thanks, yes we're making those changes. Will notify once the changes are done. :)